wazimap_ng/cache.py

```diff
@@ -19,16 +19,18 @@
 theme_key = "etag-Theme-profile-%s-%s"
 location_theme_key = "etag-Location-Theme-%s"
 
+
```

Comment:
can you remove all those empty lines please?

Reply:
https://www.python.org/dev/peps/pep-0008/#blank-lines

Can we follow the pep8 convention to add extra line between functions